docs/contributing/checklist.md

```diff
@@ -0,0 +1,167 @@
+
+If your pack contains at least an automation script, the automation directory should contain the following:
+
+- Code file (i.e. `Packs/YourPackName/YourScriptName/Scripts/YourScriptName.py`): your script implementation [code](code-conventions).
+- Metadata file (i.e. `Packs/YourPackName/Scripts/YourScriptName/YourScriptName.yml`): YML file with [script metadata](yaml-file).
+- README  file (`Packs/YourPackName/Scripts/YourScriptName/README.md`): the script  documentation, mostly autogenerated (see [here](integration-docs)).
+- Unit tests file (i.e. `Packs/YourPackName/Scripts/YourScriptName/YourScriptName_test.py`): must be included to automatically test the code during the review phase. More details [here](unit-testing)
+- Unit tests data files (i.e. `Packs/YourPackName/Scripts/YourScriptName/test_data/*.json`): contain example responses from your product API and are used in [Unit tests](unit-testing).
+
+**Note**: if you use PowerShell and not Python, the extension of the code files will be `.ps1` instead of `.py`.
+
+If your integration is going to be *certified*, you also have the following requirements (optional for non-certified):
+
+- Test Playbook (i.e. `Packs/YourPackName/TestPlaybooks/playbook-YourScriptName_Test.yml`): required for nightly builds. More information [here](test-playbooks)
+
+... and of course your adherence to our best practices and [code conventions](code-conventions) will be evaluated in a stricter way.
+
+**Note**: if your pack contains both integrations and scripts, you can use a single Test Playbook to test both.
+
+## Widgets
+
+If your pack contains at least a custom widget, you'll need:
+
+- Widget JSON file (i.e. `Packs/YourPackName/Widgets/widget-YourWidgetName.json`): if exported directly from the XSOAR UI, it must be formatted with `demisto-sdk format`.
+
+## Dashboards
+
+If your pack contains at least a custom dashboard, you'll need:
+
+- Dashboard JSON file (i.e. `Packs/YourPackName/Dashboard/dashboard-YourDashboardName.json`): if exported directly from the XSOAR UI, it must be formatted with `demisto-sdk format`.
+
+
+# Checklist Table
+
+The requirements above are also summarized in the following table:
+
+|               Entity Type |                                                                          Non-certified and certified                                                                          | Certified only  |
+|--------------------------:|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|---------------------------------|
+| Pack                      | <ul><li>Pack metadata</li><li>Pack readme</li><li>Release Notes</li></ul>                                                                                                                       | <ul><li>Test Playbook</li><li>Custom Incidents (if *fetch_incidents* is supported)</li></ul> |
```

Comment:
Add here a row with `Design` and under `Certified only` mention that certified partner must implement most of the use cases from `Use Cases`